starter_templates/go/go.mod

```
@@ -0,0 +1,3 @@
+module git.codecrafters.io/go-git
```

Assuming this has no bearing on functionality, let's make this line: 

```suggestion
module github.com/codecrafters-io/git-starter-go
```